src/shared/services/addresses/addresses.service.ts

```diff
@@ -43,14 +42,14 @@ export class AddressesService {
   }
 
   private shiftAddressLinesUp(addressModel: AddressModel) {
-    if (addressModel.AddressLine1 === '') {
-      addressModel.AddressLine1 = addressModel.AddressLine2;
-      addressModel.AddressLine2 = '';
-    }
     if (addressModel.AddressLine2 === '') {
```

Comment:
I've found a little issue int the case line 1 is empty, and line 2 and 3 have a value. This fix will push up line 2 to line 1. But leave line 3 there. It won't be pushed on line 2. not sure if this might be an issue.